doc-content/kogito-docs/src/main/asciidoc/openshift/proc_kogito-deploying-on-ocp-console.adoc

```diff
@@ -0,0 +1,89 @@
+[id='proc_kogito-deploying-on-ocp-console']
+
+= Deploying {PRODUCT} services on {OPENSHIFT} using the OpenShift web console
+
+After you create your {PRODUCT} services as part of a business application, you can use the {OPENSHIFT} web console to deploy your services. As an aid to deploy your services, you can also use the {PRODUCT} Operator, which guides you through the deployment process. The {PRODUCT} Operator uses the https://github.com/operator-framework[Operator Framework] and automates many of the deployment steps for you. For example, you can provide the Operator a link to the Git repository that contains your application, and the operator can check out the project, build it (including native compilation, if applicable), and deploy the resulting services.
```

Comment:
Just out of curiosity, is there any form of include in asciidoc? I believe it's the third time I see this paragraph. :thinking:

Reply:
Yes, ascii is built around includes. This is how docs are created, as assemblies that consist of a list of included modules (topics). However, per all Red Hat modular doc standards, includes should be used in assemblies only, not in modules (topics) themselves. And small include "snippets" are supposedly discouraged due to complexities in tooling etc. Exceptions included admonitions, such as the disclaimer about Dev Preview, but even that I've not snippeted with include because of the ongoing debate in CCS. Either way, a paragraph like this wouldn't make the cut anyway because not standard enough, supposedly. Anyway, the long answer.